api/v1/services/StatsService.js

```diff
@@ -69,27 +70,33 @@ function _populateEcosystems(cb){
  */
 function _populateAttendingEcosystems(cb){
     return Attendee.query(function(qb){
-        qb.select('e.name').count('accepted_ecosystem_id as count').innerJoin('ecosystems as e', 'e.id', 'accepted_ecosystem_id')
```

Comment:
Knex would break the old stuff down into inner `SELECT` queries, and I thought that using `JOIN`s would be better